golang/truv.go

```diff
@@ -244,25 +299,11 @@ func getEmployeeDirectoryByToken(access_token string) (string, error) {
 	return string(data), nil
 }
 
-// PayrollReportRequest defines the body of the request when requesting
-// a payroll report
-type PayrollReportRequest struct {
-	AccessToken string `json:"access_token"`
-	StartDate   string `json:"start_date"`
-	EndDate     string `json:"end_date"`
-}
-
-// PayrollReportResponse defines the body of the response when requesting
-// a payroll report
-type PayrollReportResponse struct {
-	PayrollReportId string `json:"payroll_report_id"`
-}
-
 // requestPayrollReport uses the given access token to request
 // the associated payroll report
 func requestPayrollReport(access_token, start_date, end_date string) (*PayrollReportResponse, error) {
-	fmt.Println("TRUV: Requesting a payroll report be created using an access_token from https://prod.truv.com/v1/administrators/payrolls")
-	fmt.Printf("TRUV: Access Token - %s\n", access_token)
+	log.Println("TRUV: Requesting a payroll report be created using an access_token from https://prod.truv.com/v1/administrators/payrolls")
+	log.Printf("TRUV: Access Token - %s\n", access_token)
 	reportRequest := PayrollReportRequest{AccessToken: access_token, StartDate: start_date, EndDate: end_date}
```

Comment:
Actually I don't see this endpoint in our API docs. Is it ok to keep it in quickstart app? @chetvil 

We have admin product in swagger, but it's hidden from the site: https://docs.truv.com/reference/income_verification